examples/function_memory_game/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" href="public/favicon.ico" />
+    <title>Yew • Function MemoryGame</title>
+    <base data-trunk-public-url/>
```

Comment:
```suggestion
    <title>Yew • Function Memory Game</title>
    <base data-trunk-public-url />
```

Reply:
Unless it's actually supposed to be stylised as MemoryGame...

Reply:
will update